CONTRIBUTING.md

```diff
@@ -59,9 +55,28 @@ If your code requires new dependencies, be sure that those are available on all
 
 ### User experience
 
-Jamulus is used by people all over the world with different backgrounds and knowledge. Features and language should be user focused.
-The Jamulus project has a [style and tone guide](https://jamulus.io/contribute/Style-and-Tone) which you should follow to remain consistent.
-Features should be usable in the sense that they act as expected to somebody who does not have a technical background.
+Jamulus is used by people all over the world with different backgrounds and knowledge. Therefore, think user focused.
+
+- Features should be usable in the sense that they act as expected to somebody who does not have a technical background.
+- Make sure you follow the [style and tone guide](https://jamulus.io/contribute/Style-and-Tone) to remain consistent.
+
+### Submitting code
+
+- Create a Github fork of the Jamulus code and create a Pull Request when you are done. Please don't use diff/patch to send your code changes. GitHub can be confusing, so feel free to ask for help.
+- Please run a local (build/feature) test to ensure there are no errors. After opening a pull request, keep an eye on the CI checks for quality or compile issues and fix them as required.
+
+#### Updating a PR/fixing conflicts
+
+If there are conflicts with jamulussoftware/jamulus showing when you raise your pull request or are requested to "rebase" your work, do so as follows:
+
+```shell
+git checkout master ;# checkout your local master
+git pull upstream master --tags ;# refresh from jamulussoftware/jamulus (i.e. use your upstream remote name. If you haven't set it up yet, add it via 'git remote add upstream git@github.com:jamulussoftware/jamulus')
+git push --tags --force ;# update your fork's master
+git checkout - ;# back to your working branch
+git rebase master ;# replay your changes onto the latest code (and fix conflicts if needed)
```

Comment:
Fixing conflicts might not be trivial. Is there some good external ressource we could link here?

Reply:
#2558 and pgScorpio's comment?

Reply:
@pgScorpio which one helped most in your opinion?